stable/redis-ha/templates/redis-ha-statefulset.yaml

```diff
@@ -49,6 +49,20 @@ spec:
       securityContext:
 {{ toYaml .Values.securityContext | indent 8 }}
       initContainers:
+{{- if .Values.hostPath }}
+      - name: hostpath-chown
+        image: {{ .Values.image.repository }}:{{ .Values.image.tag }}
+        securityContext:
```

Comment:
Exposing the securityContext in values.yaml might be good for those who want to use the hostPath with preconfigured permissions and/or don't want the pods to use a root account.

Reply:
I'd add a variable to make this behaviour configurable. Exposing the security context won't helm much because it only works with exactly this setting (user=0, allow root) - the alternative is to disable this init container completely via option.

Reply:
Done.

Reply:
Yeah this is a suitable alternative. Thank you